tests/acceptance/conftest.py

```diff
@@ -84,7 +84,7 @@ def login(arsenic):
     top_navigation_page.log_out()
 
 
-@pytest.fixture(autouse=True, scope='module')
+@pytest.fixture(autouse=True, scope='function')
```

Comment:
I changed the scope of this fixture because every time that we run a function test the PostgreSQL should not contain any article. Moreover when we have to do assertions about the status of an article we are assuming that only one article is present in the list and it helps also to maintain the tests more independent with each other.